### ISSUE TITLE
Building placement is impossible to figure out

### ISSUE THREAD

**Issue by SpeedSlider** [ldtteam/minecolonies#2696]
### Minecolonies version

version 0.8.9646 (1.12.2 included with ATM3 Modpack)

### Expected behavior

- Building tool to help placing and building various buildings.

### Actual behaviour

- Default seems to be for just placing caravan or supply ship. It also seems like it would be useful for various non-building functions like roads and walls. There is no manual so it is unclear if you are supposed to place the key block for each building or keep it in your inventory and use the building tool which I only just thought of as an option after reading the various open issues. When placing the block for each building construction tape goes up but there is no indication on which way the building will face and again trying to use the building wand does not give a preview of that build even when cycling through all the gui menu options. Additionally, the construction tape does not represent the area of the build and the unaffected areas are left after the build is complete.

I have tried several methods in game to get previews of the buildings before approving the build and none seem to work for me. The area of affect is also outside the construction tape zone so that can't even be used to verify you won't impact other buildings or builds around the area. Even if there is plenty of area and the building is 2-3 blocks away from other buildings it does not guarantee that the builder will not remove things like walls or other structures in order to stand several blocks away and build the structures. I have build 6 level one structures and they either do not face the way I would like them to or do not build at the level I expect them to which means they are impossible to build as a cohesive town and some are dug into the ground while others are raised above it.

Bottom line: create a manual for new players. The mod is too difficult for me to figure out and I don't appear to be the only one based upon limited research and YouTube videos. It was worth a try, I guess, but looks like I will be scrapping the buildings and going back to building my own and using machines for various functions instead of minecolonies.

### Steps to reproduce the problem

1. Be a noob
2. try to figure out how the buildings work and if there is a hierarchy to which ones you need first
3. try using the wand for building previews after placing the block because that is how the supply caravan worked.
4. realize that if you place and pick up the block after assigning a build and realizing it is not where you want it based upon the builders digging you now have several ques for the same building in the builders hut.
5. realize you are too noob to work around the poorly conceived construction tape and building orientations so you write this issue and give up on the mod.
6. also realize that it wouldn't be that hard to include and arrow or something to indicate where the front of the building is going to be located.
7. no clue on a recommendation for how to determine y level build if the building tool gui doesn't give you a building preview.

**Comment by SpeedSlider** [comment]
The beginning of my next stream will show my placement woes.
https://www.twitch.tv/speedslider

**Comment by Raycoms** [comment]
Hi, it's quite easy actually. When you right click with the build tool and having the block in your inventory, will render the building, if not, in the top left there will be a dropdown list where you can select the building you want to place. This should render it. The construction tape represents the biggest of the building of the current style so level 5 usually, so it might be slightly bigger than what the building looks like on the first level. (The tool also lets you select the level 5).
I sent the ATM3 creators already a message to update minecolonies since they are a lot of builds behind of minecolonies.

Check out our newest video, it shows how to use it properly: https://www.youtube.com/watch?v=JDVHmcc8-L0&lc=z22tu5l5hvauyr1czacdp4312nduruqzo10r33l0dy1w03c010c

**Comment by SpeedSlider** [comment]
Tried that… not to easy for a noob apparently, the tape doesn’t encompass the whole area impacted as the building digs and builds outside the taped area… but no worries… Gotchya… don’t need to play the mod not sorry it is being removed from my favorite mod pack in the next update. Still could use a decent manual or tutorial videos, yes I tried watching some from the minecolonies team, guess I am just an idiot.

Sent from Mail for Windows 10

**Comment by Raycoms** [comment]
The video I linked I uploaded just today because we noticed there are not a lot of videos on the matter, you should try to check it out.

**Comment by SpeedSlider** [comment]
Nah… you wanted to close my issue too quickly and I already spent too much time trying to find videos and tutorials. I understand your issue with a modpack being several versions behind, but perhaps some real conversation other than just being dismissive and blaming a modpack for not having an up to date version would have been a better approach from my perspective. I mostly just left my feedback on your GitHub incase there were suggestions for better implementations that haven’t been considered before as a last ditch before giving up.

At least you responded quickly.

Sent from Mail for Windows 10

**Comment by Raycoms** [comment]
I closed this issue because it wasn't an issue on the implementations but because of bad usage. We have a ton of players, also on our discord which have no issues using this tool, that's why I linked you the video as well.

**Comment by SpeedSlider** [comment]
I couldn’t get it to work like the video below… I understand it is 1.11.2 but after watching this video and trying to build the warehouse and delivery service without any better success I just realized I am too noob to figure it out. No need for further follow-up since I have decided not to pursue the mod any further; the progression and citizens pace are too slow for me even if I could figure out how to properly place the buildings. I do understand you have a strong following with many more intelligent players than myself and my opinion will not harm your success and that is not my intent. Happy modding.

https://www.youtube.com/watch?v=DVGGDUXbTOY&t=11s

Sent from Mail for Windows 10
